mentees/README.md

```diff
@@ -0,0 +1,264 @@
+## **Become a Mentee -- Draft**
```

Comment:
Before we merge, this should have draft removed from the title, correct? Or is the content going to be left unfinished (such as the remuneration section) when we publish this change?